### ISSUE TITLE
Add new Insurance Co Fails on dev branch

### ISSUE THREAD

**Issue by tmccormi** [LibreEHR/LibreEHR#251]
ERROR: insert failed: INSERT INTO insurance_companies ( id, name, attn, cms_id, ins_co_type, x12_receiver_id, x12_default_partner_id ) VALUES ( 2, 'Insurance Company', '', '12345', '1', '', '' )

Error: Unknown column 'ins_co_type' in 'field list'

/var/www/LibreEHR/interface/practice/ins_search.php at 180:sqlInsert

**Comment by tmccormi** [comment]
Database.sql says the column name should be:
`ins_type_code` tinyint(2) default NULL,

but the following programs:
custom/qrda_functions.php
interface/practice/ins_search.php

Say "ins_co_type" @aethelwulffe  appears to be the last person to touch those files

**Comment by aethelwulffe** [comment]
This was from the freeb removal.  I will investigate where either field name exists (as well as freeb_type) and go from there.

**Comment by aethelwulffe** [comment]
5 files vs. 2 files.  ins_type_code wins.  The ins_co_type is a version from an openemr grab (or presumably parallel effort assuming the original author did it) of the removal of freeb.  In any case, our version existed first, so I am not sure (at the moment) how the other two files came up with those differences.  Best to look at the blame for those.  I am pretty sure that we are worried about the patient's insurance type, vs the type of insurance company...those are not mutually exclusive, and I prefer the other term.
#253

**Comment by aethelwulffe** [comment]
Looks like I had actually stomped on my own code at some point.  I had used ins_co_type during a code array update.  That array was mixed with the freeb code, etc...  Later, in getting rid of freeb, I used a different name.  The should all be synced now, with the array being named ins_type_code_array as well, so everything matches.
OpenEMR has adopted the ins_co_type, and we use ins_type_code as the replacement for freeb_type.
